source/partials/valid_test_account_numbers.erb

```diff
@@ -0,0 +1,16 @@
+<aside class="notice">
+When testing payments in the sandbox environment please make sure you use valid sort codes and
+account numbers as these are validated to ensure they accept Faster Payments prior to being sent.
+</aside>
+
+Below are some valid sample details you can use:
+
+|Account Number|Sort Code|
+|--------------|---------|
+|87889196      |166051   |
+|99993193      |166051   |
+|00004588      |203002   |
+|00027944      |203002   |
+|58110244      |204514   |
+|60282377      |204514   |
```

Comment:
This is just what was there before but it could use a revamp.

There should be no ambiguity about this.
Also have in mind we deal with Starling and Modulr (if it makes any difference).

Dean sent this from Starling docs:

```
Outbound payments are initiated through the API. The simulator will accept payments to any FPS addressable sort code. If the sort code is operated by Starling Bank, the destination account must also exist for the payment to be accepted (for example transferring between two addresses for your accounts).
To help with testing payments to accounts outside of Starling, the following accounts are valid:
Sort Code	Account Number
203002	00004588
203002	00027944
204514	58110244
204514	60282377
166051	99993193
166051	87889196

The following accounts are invalid:

Sort Code	Account Number
203002	00004900
203002	00028000
204514	58110250
204514	60282380
166051	99993200
166051	87889200
```